### DISCUSSION TITLE
How does the Pixelfed Live feature work?

### DISCUSSION THREAD

**Discussion by jaschaurbach** [pixelfed/pixelfed#4819] [Support] [answered]
I am curious - in the documentation is a section **[Livestreaming (Pixelfed Live)](https://docs.pixelfed.org/running-pixelfed/optional-features/livestreaming/)**.

The feature was advertised by the [Pixelfed Mastodon Account](https://mastodon.social/@pixelfed/108356310744708214)
I wonder as well which app the live feature got merged (see this [Pixelfed Post](https://mastodon.social/@pixelfed/109319130810399820)): Android or iOS?

In the source I finde many references of the Live feature, e.g. [here](https://github.com/pixelfed/pixelfed/blob/7ac1e398b8f340eb592151bd6875a9329269333f/routes/web.php#L670) but [here it looks like](https://github.com/pixelfed/pixelfed/blob/7ac1e398b8f340eb592151bd6875a9329269333f/resources/assets/js/spa.js#L128) the feature is commented out and the commented components are not in the repository.

I really would love to use this feature with OBS - how do I obtain a sreamkey? 

In general it would be great to finally get more than installation and config of the feature but a how to really activate and use it - from app and/or OBS and how to view these lives.

Thank you for all the great work you have done with pixelfed.

**Comment by dansup** [comment]
We are in the process of refactoring this feature, I will let you know when the updated version is available and link you to the updated docs!

**Comment by jaschaurbach** [comment, marked as answer]
I just do not care anymore.
